iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothubservices/FileUploadAndroidRunner.java

```diff
@@ -47,4 +47,10 @@ public static void setup() throws IOException
     public void uploadToBlobAsyncSingleFileZeroLength()
     {
     }
+
+    @After
+    public void labelSnapshot()
+    {
+        reportHelper.label("Stopping App");
```

Comment:
do you know what is the effect of adding this label?  and if we are adding the same label in every class...how is it going to help them in debugging?  BTW, @After is executed at the end of the test class...so not really stopping the app itself...

Reply:
https://docs.microsoft.com/en-us/appcenter/test-cloud/preparing-for-upload/espresso

Reply:
The "Stopping app" string is inconsequential for our purposes. The important bit here from the above article: 

"recommended practice is to have a call to label in the @After method, this includes a screenshot of the app final state in the test report. The screenshot is taken, even if a test is failing, and often provides valuable information as to why it does so."